components/studio/test/shared/studio-enter.ps1

```diff
@@ -8,7 +8,7 @@ $env:HAB_NOCOLORING = "true"
 # Await has trouble parsing non-ascii glyphs
 $env:HAB_GLYPH_STYLE = "ascii"  
 $exit_code = 0
-$studio_name = "/hab/studios/studio-internals-test"
+$studio_name = "c:\hab\studios\studio-internals-test"
```

Comment:
This is fine our purposes here and indeed almost any scenario. I always like to use:
```
$ExecutionContext.SessionState.Path.GetUnresolvedProviderPathFromPSPath("/some/path/I/want/to/cannonacalize")
```
This will always transform the given string to a Windows friendly absolute path whether or not the path actually exists and applicable to one's current context. In otherwords, if you were in the `d:\` drive, this would resolve to the `d` drive.

Note there is also the simpler and more intuitively named `Resolve-Path` which does the same thing but requires the given path to exist. If I know or expect the path to already exist, I prefer this command.